man/systemctl.xml

```diff
@@ -2003,6 +2003,11 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         <varname>Triggers=</varname> in <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>
         for more information about triggering units.</para>
 
+        <para><literal>restart-dependencies</literal> may only be used with
+        <command>systemctl start</command>. In this mode, dependencies of
+        the specified unit will receive restart propagation, as if a restart
+        job had been enqueued for the unit.</para>
```

Comment:
so i actually didn't document this on purpose, because it sounded too specialist to me, i wanted to keep this as an undocumented half-internal feature. But actually, thinking about it it might be ok, there might be other uses for this. hnece commit looks ok.